.github/workflows/design-system-governance.yml

```
@@ -0,0 +1,523 @@
+          fetch-depth: 2
+          
+      - name: 🔍 Check Changes
+        id: check-changes
+        run: |
+          # Check if governance-related files changed
+          if git diff --name-only HEAD~1 | grep -E "(design-tokens|components|lib/design-system|scripts)" > /dev/null; then
+            echo "should-run=true" >> $GITHUB_OUTPUT
+          else
+            echo "should-run=false" >> $GITHUB_OUTPUT
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)